ci-operator/config/openshift/aws-efs-csi-driver-operator/openshift-aws-efs-csi-driver-operator-release-4.14.yaml

```diff
@@ -103,6 +103,10 @@ tests:
   commands: make verify
   container:
     from: src
+- as: e2e-aws-ovn-upgrade
```

Comment:
AWS EFS CSI driver is installed via OLM, i.e. it's not used anywhere in this job. I think we don't have any workflow that install the driver, upgrades the cluster and then runs some tests with the driver. Please remove the jobs, also from openshift-aws-efs-csi-driver repo.